### PR TITLE
chore: update dashboard swagger document

### PR DESCRIPTION
**Pull request by datphamcode295**
#### What's this PR do?

- [x] update dashboard swagger document

### CODE REVIEW

File: pkg/handler/dashboard/dashboard.go [hunk 1]
```diff
@@ -476,11 +482,12 @@ func (h *handler) GetActionItemSquashReports(c *gin.Context) {
 // GetSummary godoc
 // @Summary Get the summary audit info for projects
 // @Description Get the summary audit info for projects
+// @id getSummary
```
Comment (hieuphq) [inline pkg/handler/dashboard/dashboard.go:485]:
```suggestion
// @id getProjectSummary
```

File: pkg/view/dashboard.go [hunk 2]
```diff
@@ -998,3 +1020,25 @@ func ToWorkSummaries(eers []*model.EmployeeEventReviewer) []WorkSurveySummary {
...
+
+type ResourceUtilization struct {
+	Date      time.Time `json:"date"`
+	Staffed   int       `json:"staffed"`
+	Internal  int       `json:"internal"`
+	Available int       `json:"available"`
+} // @name ResourceUtilization
+
+func ToReSourceUtilizations(data []*model.ResourceUtilization) []ResourceUtilization {
+	rs := make([]ResourceUtilization, 0)
```
Comment (hieuphq) [inline pkg/view/dashboard.go:1032]:
```suggestion
	rs := make([]ResourceUtilization, 0)
```
must modify following golang practice

**Review by hieuphq — CHANGES_REQUESTED** [review]

File: pkg/view/dashboard.go [hunk 3]
```diff
@@ -998,3 +1020,25 @@ func ToWorkSummaries(eers []*model.EmployeeEventReviewer) []WorkSurveySummary {
...
+
+type ResourceUtilization struct {
+	Date      time.Time `json:"date"`
+	Staffed   int       `json:"staffed"`
+	Internal  int       `json:"internal"`
+	Available int       `json:"available"`
+} // @name ResourceUtilization
+
+func ToReSourceUtilizations(data []*model.ResourceUtilization) []ResourceUtilization {
+	rs := make([]ResourceUtilization, len(data))
```
Comment (hieuphq) [inline pkg/view/dashboard.go:1032]: sai rồi

**Review by hieuphq — CHANGES_REQUESTED** [review]

File: pkg/handler/dashboard/dashboard.go [hunk 4]
```diff
@@ -709,14 +721,15 @@ func (h *handler) GetWorkUnitDistributionSummary(c *gin.Context) {
 // GetWorkUnitDistribution godoc
 // @Summary Get work unit distribution data for dashboard
 // @Description Get work unit distribution data for dashboard
+// @id getWorkUnitDistribution
```
Comment (hieuphq) [inline pkg/handler/dashboard/dashboard.go:724]:
```suggestion
// @id getWorkUnitDistributionList
```

File: pkg/handler/dashboard/dashboard.go [hunk 5]
```diff
@@ -610,13 +619,14 @@ func (h *handler) GetEngagementInfo(c *gin.Context) {
 // GetEngagementInfoDetail godoc
 // @Summary Get engagement dashboard
 // @Description Get engagement dashboard
+// @id getEngagementInfoDetail
```
Comment (hieuphq) [inline pkg/handler/dashboard/dashboard.go:622]:
```suggestion
// @id getEngagementInfoDetailList
```

**Review by hieuphq — CHANGES_REQUESTED** [review]

File: pkg/handler/dashboard/dashboard.go (hunk 5 above)
Comment (datphamcode295) [reply pkg/handler/dashboard/dashboard.go]: updated

**Review by hieuphq — APPROVED** [review]

**Review by hieuphq — APPROVED** [review]

**Pull request merged**